utils/build-script

```diff
@@ -596,26 +593,6 @@ class BuildScriptInvocation(object):
         if args.build_swift_static_sdk_overlay:
             impl_args += ["--build-swift-static-sdk-overlay"]
 
-        if not args.build_linux:
-            impl_args += ["--skip-build-linux"]
-        if not args.build_freebsd:
-            impl_args += ["--skip-build-freebsd"]
-        if not args.build_cygwin:
-            impl_args += ["--skip-build-cygwin"]
-        if not args.build_osx:
```

Comment:
Does this really not do anything? I am pretty sure I have used these in the past bit.

Reply:
If invoked originally from the Python `build-script`, it hasn't done anything in this bash script in many years, as it's handled in Python, #2880. As a result, the actual implementation of this flag in this bash script was removed last year in the aforementioned pull #26495.